### ISSUE TITLE
Force enable transition={true} for drawer doesn't work

### ISSUE THREAD

**Issue by robotpapier** [skeletonlabs/skeleton#2085]
### Current Behavior

No animation when drawer opens and closes

### Expected Behavior

to be animated

### Steps To Reproduce

As mentionned in docs : [https://www.skeleton.dev/docs/transitions#force-enable](url)
Even if $prefersReducedMotionStore is true, 
adding transition={true} should force the animation

but it's not

```
<Drawer transition={true}>
    <h2 class="p-4">Navigation</h2>
	<hr />
    <Navigation />
</Drawer>
```

**Comment by robotpapier** [comment]
Re,
it was a typo in the doc
The correct prop is : transitions={true} instead of transition={true}

**Comment by endigo9740** [comment]
> Re, maybe a typo ? At least for drawer component, the correct prop is : transitions={true} instead of transition={true}

Yep, that seems to be the case. You can see the prop in the component source here:
https://github.com/skeletonlabs/skeleton/blob/master/packages/skeleton/src/lib/utilities/Drawer/Drawer.svelte#L71

Should be an easy fix, thanks for the heads up!

@Mahmoud-zino if you wish to get this, go for it.
